server/src/actions/createMessage.ts

```diff
@@ -1,17 +1,18 @@
 import { doc, setDoc } from '@firebase/firestore'
 import { messages } from '../utilities/firebaseInit'
+import { Message } from '../types/Message'
 
-export const createMessage = async (userId: string, msgId: string, msgContent: string, lat: number, lon: number, geohash: string, timeSent: Number) => {
-   const newMsgRef = doc(messages, msgId) 
+export const createMessage = async (msg:Message) => {
+   const newMsgRef = doc(messages, msg.msgId)
    const docData = {
```

Comment:
Honestly not a super big deal, but do you think you could make docData more type safe by adding Message type to it?
Something like this `docData: Message = {..}` might work!

Reply:
That sounds like a good Idea. Will do

Reply:
should be done now